src/templates/app/user/metatrader.jsx

```diff
@@ -308,33 +317,52 @@ const Metatrader = () => (
-                                    <SubmitButton
-                                        is_centered
-                                        type='submit'
-                                        id='btn_submit_new_account'
-                                        text={it.L('Create account')}
-                                        attributes={{ action: 'new_account' }}
-                                        custom_btn_text={it.L('Back')}
-                                        custom_btn_id='btn_back'
-                                        custom_btn_class='button-secondary'
+                                    <input type='hidden' id='txt_name' />
+                                    <p id='msg_form' className='invisible error-msg' />
+                                    <FormRow
+                                        is_two_rows
+                                        type='password'
+                                        id='txt_main_pass'
+                                        label={it.L('Password')}
                                     />
+                                    <div className='gr-padding-10 center-text'>
+                                        <ResetPasswordButton />
+                                        <button
+                                            id='btn_submit_new_account'
+                                            type='submit'
+                                            action='new_account'
+                                            className='button button-secondary'
+                                        >{it.L('Next')}
+                                        </button>
+                                    </div>
+                                </div>
+                            </div>
+                            <div id='view_3' className='gr-row invisible'>
+                                <div className='container gr-8 gr-12-m center-text'>
+                                    <p id='password_reset_error' className='invisible error-msg center-text' />
+                                    <ResetPasswordButton />
+                                </div>
+                            </div>
+                            <div id='view_4' className='gr-row invisible'>
+                                <div className='container gr-8 gr-12-m'>
+                                    <h2>{it.L('Reset password')}</h2>
+                                    <p className='notice-msg center-text'>{it.L('Please check your email for further instructions.')}</p>
                                 </div>
                             </div>
                         </div>
                     </form>
                 </div>
                 <div id='frm_manage_password'>
                     <div className='gr-row'>
-                        <div className='gr-6 gr-12-m flex'>
+                        <div className='gr-12 gr-12-m flex'>
```

Comment:
We can remove `gr-12-m` since the number is the same for desktop and mobile

Reply:
Thanks for the feedback, but I had to bring the gr-6 back due to design changes. :(